datasets/imagenet_download.py

```diff
@@ -23,7 +23,7 @@ def imagenet_prepare_val():
   # Create folders and move files into those
   for co,dir in enumerate(labels):
     os.makedirs(Path(__file__).parent.parent / "datasets/imagenet/val" / dir, exist_ok=True)
-    os.replace(Path(__file__).parent.parent / "datasets/imagenet/val" / images[co], Path(__file__).parent.parent / "datasets/imagenet/val" / dir / images[co], exist_ok=True)
```

Comment:
related?

Reply:
yeah, was fixing syntax errors that made this not work - exist_ok is not a param. can move to a separate PR if you'd like.

Reply:
oh that's fine